extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -192,7 +193,7 @@ export class SchemaCompareDialog {
 
 			this.targetTextBox = this.view.modelBuilder.inputBox().withProperties({
 				value: this.schemaCompareMainWindow.targetEndpointInfo ? this.schemaCompareMainWindow.targetEndpointInfo.packageFilePath : '',
-				width: 275,
+				width: 280,
```

Comment:
reason for this?

Reply:
To reduce this gap:
![image](https://user-images.githubusercontent.com/57200045/115759311-f51f9480-a354-11eb-8bc4-5fdd76f05552.png)